### ISSUE TITLE
Projects and milestones so we can plan dead lines for certain objectives

### ISSUE THREAD

**Issue by elreydetoda** [49thSecurityDivision/website#5]
I think this would be awesome so we can stay all organized on here and if anyone else wants to help out, then they can/others can see how we work on things and that might get them to help out.

**Comment by ethomp-dev** [comment]
Yes, I will definitely work on planning things out for the next semester.

**Comment by elreydetoda** [comment]
Awesome, just when you get a chance it will be here =)
